backend/package.json

```diff
@@ -28,8 +28,10 @@
   },
   "devDependencies": {
     "concurrently": "3.4.0",
+    "jasmine": "2.6.0",
```

Comment:
We should use Jest instead of Jasmine. Jest has really nice TypeScript support and amazing testing capabilites for React. It's super mature by now.